PackageInstallations_Troubleshoots.Rmd

```diff
@@ -0,0 +1,124 @@
+---
```

Comment:
This is an Rmd file, is it needed (if we have qmd)?

Reply:
Oh yes, I forget to removed this, now I only added the qmd

Reply:
resolve this discussion when this has been dealth with.